example/src/app/wyf/wyf.page.html

```diff
@@ -0,0 +1,7 @@
+<ion-content [fullscreen]="true">
+  <!-- Add this component to your body -->
+  <!-- Retreive your situm api-key here https://dashboard.situm.com/accounts/profile -->
+  <!-- Follow this guide (https://situm.com/docs/sdk-cartography/#building-identifier)
+    to find out the identifier of the building you want to display -->
+  <map-view #mapView> </map-view>
```

Comment:
E as propiedades? Parece que as poblas en código, pero non creo que vaia a ser o máis habitual.
Paréceme cómodo para nós, quizais poidas poñer aquí placeholders e cambialos en código, tal cual o tes.
Dese xeito calquera pode ver como é a forma "habitual" de facelo e tamén pode ver que se cambian en código os valores, por movidas do noso repo.

Reply:
Mmm certo, o mellor e deixar as duas formas.